testing/constraints-3.9.txt

```diff
@@ -1 +0,0 @@
-sqlalchemy>=1.4.13,<2.0.0
```

Comment:
For 3.9, does it make sense to set a specific version, much as we did for 3.8?

if setup.py etc have the right minimum version of python, then it prolly doesn't matter. just trying to think of whether I am missing anything.